generate/generate.go

```diff
@@ -782,6 +782,25 @@ func (s *service) generateHelperFuncs(a *API) {
 			pn("		return nil, -1, err")
 			pn("	}")
 			pn("")
+			pn("    if l.Count == 0 {")
+			pn("		// look	inside projects ")
+			pn("		p.p[\"projectid\"] = \"-1\"")
```

Comment:
Shouldn't we do a small `if` here checking if we actually are in a function that has a `projectid` parameter?